### PR TITLE
ARP: add tutorial

### PR DESCRIPTION
**Pull request by miekg**

### CODE REVIEW

File: website/content/tutorial-arp.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
+---
+title: Tutorial for ARP
+weight: 10
+---
+
+In this tutorial we deploy MetalLB in a (ARM) cluster and announce a load-balanced IP using ARP. We
+assume you have a bare metal [cluster
+running](https://blog.hypriot.com/post/setup-kubernetes-raspberry-pi-cluster/).
+
+The nice thing of (abusing) the ARP protocol is that you don't need any fancy network hardware at
```
Comment (danderson) [inline website/content/tutorial-arp.md]: I don't think we should say abusing here, from the network's POV we are behaving perfectly correctly: we are saying that we own an IP, and when the network sends packets to us, we handle them correctly.

File: website/content/tutorial-arp.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+Here is the outline of what we're going to do:
+
+1. Install MetalLB on the cluster,
+1. Configure MetalLB to announce using the ARP and give it some IP addresses to manage,
+1. Create a load-balanced service, and observe how MetalLB sets it up.
+
+## Cluster addresses
+
+The Raspberry PI cluster we have build using 192.168.1.0/24 as the address space. The main router
+is configured to hand out (DHCP) address in 192.168.1.100-150. As the load-balanced IP addresses need
```
Comment (danderson) [inline website/content/tutorial-arp.md]: DHCP addresses. And I think you can remove the parentheses.

File: website/content/tutorial-arp.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+I1217 10:19:05.005671       1 arp.go:96] Request: who-has 192.168.1.240?  tell 192.168.1.1 (b4:75:0e:63:b2:20). reply: 192.168.1.240 is-at b8:27:eb:86:e2:85
+I1217 10:19:05.105780       1 arp.go:96] Request: who-has 192.168.1.240?  tell 192.168.1.1 (b4:75:0e:63:b2:20). reply: 192.168.1.240 is-at b8:27:eb:86:e2:85
+I1217 10:19:05.235623       1 arp.go:96] Request: who-has 192.168.1.240?  tell 192.168.1.1 (b4:75:0e:63:b2:20). reply: 192.168.1.240 is-at b8:27:eb:86:e2:85
+```
+
+MetalLB is sending out unsolicited ARP responses and replies to ARP requests with the MAC address of
+the node that has one the master election and using the first address of the assigned range
+(192.168.1.240).
+
+When you curl <http://192.168.1.240> you should see the default nginx page: "Welcome to nginx!"
```
Comment (danderson) [inline website/content/tutorial-arp.md]: linkify in the markdown syntax?

**Comment by miekg** [comment]
PTAL

don't understand what you mean with "linkify in the markdown syntax?", used backticks now.

**Pull request merged**